tests/runtime/variable

```diff
@@ -49,8 +49,15 @@ NAME 32-bit tracepoint arg
 PROG tracepoint:random:urandom_read { $i = args->got_bits; printf("bits: %d\n", $i); if ($i == 24) { exit() } }
 EXPECT bits: 24
 TIMEOUT 5
+MAX_KERNEL 5.16
 AFTER dd if=/dev/urandom bs=3 count=1
 
+NAME 32-bit tracepoint arg mode
+PROG tracepoint:syscalls:sys_enter_access /comm == "tp_access"/ { if (args->mode == 2) { printf("mode: %d\n", args->mode);  exit(); } }
+EXPECT mode: 2
+TIMEOUT 5
+AFTER ./testprogs/tp_access
```

Comment:
If you used a different syscall than `access` (e.g. `openat`), you could use `AFTER ./testprogs/syscall openat [<ARGS>]` and wouldn't need to write another testprog.